environment.yml

```diff
@@ -0,0 +1,18 @@
+name: fink-client
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - python>=3.9
+  - coverage>=4.2
+  - coveralls
+  - codecov
+  - python-confluent-kafka=1.9.2
```

Comment:
Following #162 , this package should be upgraded to
```
confluent-kafka==2.0.2
```
to allow with python 3.11

Reply:
`confluent-kafka=2.0.2` is not available on conda forge yet, [see](https://anaconda.org/conda-forge/python-confluent-kafka)
Let's wait a week or so :)

Reply:
or may be I could actually use `python-confluent-kafka>=1.9.2` so that conda can handle the different versions by itself?

Reply:
I have tried with `python-confluent-kafka>=1.9.2` and 
actually `confluent-kafka=2.0.2` is install when running the `pip install fink-client`.
So I think it's all fine also for the future.
I am going to commit that.

Reply:
great, thanks!